02-microarray/00-intro-to-microarray.Rmd

```diff
@@ -6,3 +6,67 @@ output:
+
+<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/13fd8c2d8469f2ac35e1608b4a952a5f5fe9698c/components/figures/microarray-overview.jpg" width=400>  
+
+[@microarray-video]
+
+As with all experimental methods, microarray has strengths and limitations that you should consider in regards to your scientific questions. 
+
+### Microarray data **strengths**:  
+
+- Less expensive than RNA-seq - can afford more replicates and get higher statistical power [@Tarca2006]. 
+- Microarray has a faster turn-around than RNA-seq [@LCSciences2014].
+
+### Microarray data **limitations**:  
+
+- If a transcript doesn't have a probe designed to it, you won't get it measured; can't be used for transcript discovery [@Mantione2014]. 
+- A chip's probe designs are only as up to date as the genome annotation at the time it was designed [@Mantione2014].
+- As is true for all techniques that involve nucleotide hybiridization (RNA-seq too); microarray probes come with some biases depending on their nucleotide sequence composition (like GC bias). 
+
+To help address these probe nucleotide composition biases, refine.bio uses [SCAN](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3508193/pdf/nihms401888.pdf) normalization methods prior to your downloads [@Piccolo2012].
+
+## About quantile normalization 
+
+Microarray data is generally processed in groups of chips - this can lead to [batch effects](https://en.wikipedia.org/wiki/Batch_effect#:~:text=In%20molecular%20biology%2C%20a%20batch,of%20interest%20in%20an%20experiment). 
+To minimize this, all refine.bio microarray data downloads come [quantile-normalized](https://en.wikipedia.org/wiki/Quantile_normalization) which enables more confident comparisons of expression levels between experiments.
+See the refine.bio docs for more about the microarray process steps including the [quantile normalization](http://docs.refine.bio/en/latest/main_text.html#quantile-normalization).
+
+## More resources on microarray technology: 
+
+- [Getting started in gene expression microarray analysis](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000543) [@Slonim2009].
+- [Microarray and its applications](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3467903/) [@Govindarajan2012].
+- [An tutorial review of microarray data analysis](http://www.ub.edu/stat/docencia/bioinformatica/microarrays/ADM/slides/A_Tutorial_Review_of_Microarray_data_Analysis_17-06-08.pdf) [@Sanchez2008].
+- [Analysis of microarray experiments of gene expression profiling](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2435252/) [@Tarca2006].
+- [Some nice summary slides about microarray data analysis from Emory](http://web1.sph.emory.edu/users/hwu30/teaching/bioc/GE1.pdf) [@intro-microarray-slides].
+
+### Why doesn't the gene I care about show up in this microarray dataset?
+
+If a gene is not detected in any of the samples in a set through our processing, it is still kept in the `Gene` column, but it will be reported as `0`.
```

Comment:
Expanding a bit: this was not my understanding of how merges happen in refine.bio. I thought (could be wrong) that if a gene was not on a chip, it would not appear in the dataset at all (no zero, no geneid). If two chips are merged where one has the gene and the other does not, I thought we did an inner join: so the gene would not show up in the combined set. This is based on my reading of http://docs.refine.bio/en/latest/faq.html#why-do-i-get-a-limited-number-of-genes-back-when-i-aggregate-samples-from-different-platforms

Reply:
I've been confused on how to know what is kept and what is not in refine.bio datasets. Had similar confusions with RNA-seq. But I agree that that paragraph seems to suggest they are dropped.

Reply:
I will do some downloading and poking at some refine.bio datasets and report back

Reply:
There are probes removed from the original chip design, but some of those might be background and other control probes. I took a look at a Affymetrix Human Gene 1.1 ST Array which originally has [`33297` probes listed in the GEOplatform](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL11532) but has `21661` probes listed in the refine.bio download. 

If I remember correctly, I think outdated probes are removed too based on [BrainArray's updated CDF files](http://brainarray.mbni.med.umich.edu/Brainarray/Database/CustomCDF/). That also adds to the 12k probes removed in this example? And because I've used an ST chip as an example, it will probably have more probes removed from the original CDF than other chip designs.

Different chips do have different numbers, so the number of genes reported isn't based on whatever the chips are being mapped to. But the same chips have the same number of probes reported over any datasets they are being used in (based on only a couple datasets I downloaded) so I don't think there are signal cutoffs.

Reply:
I guess to summarize, my questions from here are, 1) is it BrainArray's CDFs (and relatively nothing else) that determines what probes/ensembl genes are reported in refine.bio and 2) What of this information do we think users need to know in this context. I don't really want to go on too much of a probe design bunny trail.

Reply:
I can download BrainArray CDFs and see if this lines up.

Reply:
I found the numbers, and it looks like its pretty directly to whatever version of brainarray was used. hugenest11 has `21661` probes reported by brainarrray in version 22 (which is what refine.bio said was used for the dataset). http://brainarray.mbni.med.umich.edu/Brainarray/Database/CustomCDF/22.0.0/ensg.asp

So I'll make microarray section header reflect this but try not to go on a bunny trail.